openid-4-verifiable-presentations-1_0.md

```diff
@@ -2204,9 +2229,9 @@ The following exchange protocol values are defined by this specification:
 
 ## Request {#dc_api_request}
 
-The Verifier MAY send all the OpenID4VP request parameters to the Digital Credentials API (DC API).
+The Verifier MAY send an Authorization Request, as defined in (#vp_token_request), to the DC API.
```

Comment:
```suggestion
The Verifier MAY send a Request, as defined in (#vp_token_request), to the DC API.
```

we've been avoiding to call this authorization request, since technically we allow different grants, including authorization code grant